pgx-version-updater/src/main.rs

```diff
@@ -67,7 +73,7 @@ struct UpdateFilesArgs {
 }
 
 // List of directories to ignore while Walkdir'ing. Add more here as necessary.
-const IGNORE_DIRS: &'static [&'static str] = &[".git", "target"];
+const IGNORE_DIRS: &[&str] = &[".git", "target"];
```

Comment:
@workingjubilee For the uninitiated, why can static be removed here? Because it's `const`?

Reply:
Correct, `const CONST: &T` **must** be `const CONST: &'static T`, effectively.